maxim/logger/together/utils.py

```
@@ -0,0 +1,227 @@
+                        for content_item in content:
+                            if isinstance(content_item, dict) and content_item.get("type") == "image_url":
+                                image_url_data = content_item.get("image_url", {})
+                                image_url = image_url_data.get("url", "")
+                                if image_url:
+                                    generation.add_attachment(UrlAttachment(
+                                        url=image_url,
+                                        name="User Image",
+                                        mime_type="image"
+                                    ))
+        except Exception as e:
+            scribe().warning(f"[MaximSDK] Error adding image attachments: {e}"),
```

Ruff (0.11.9)

Trailing comma on bare tuple prohibited

(COM818)